chapters/6.md

```diff
@@ -26,6 +26,24 @@ This concerns the reference C++ implementation of Monero. Please note there are
 
 ## Hashing
 
+Hashes are one-way mathematical functions that take a variable length input of any size and produces a unique fixed length output called a hash value. This function is considered as 'one way' as it is very difficult to find the original input of a hash value. This is unless you try to brute force guess all existing inputs to find a matching hash value.
+
+(If you are interested in the security of the hash function, check out this video: Security of SHA-256)
```

Comment:
Can you please include the link of the video too? I may add a link to ebook and pdf :)